conformance.yaml

```diff
@@ -998,7 +998,7 @@
   outputs:
     ga4ghMd5.value:
       type: File
-      value: {md5sum: '24b6378740d9662c8f464364b10a133a'}
+      value: {md5sum: 'd3d7db14b8d14dcf8232f921a71d1b08'}
```

Comment:
What did you do that would cause the MD5 hash to change here? The tool you're replacing is supposed to just spit out the MD5 of the input: https://github.com/briandoconnor/dockstore-tool-md5sum/blob/123bde0d30035360c45ebfdebcc4c90669cf958c/bin/my_md5sum#L4

So one of these hash values actually is the MD5 of the input, and the other is not, or else this commit changes what the input is. Or else the test was just fundamentally wrong to start.

What are the actual inputs supposed to be here?

Reply:
Oooh I think these hashes here are the (sha1 I guess) hashes of the *files containing* the hashes of the workflow inputs. And they changed because the new WDL doesn't cut off and throw away the filename that `md5sum` prints, like the old one did inside the Docker container.

I think you should more closely replicate the functionality of the old tool and cut off and throw away the filename from the `md5sim` hashes, and then you won't have to change the expected result.